### PR TITLE
Tweaks to dev instructions.

### PR DESCRIPTION
**Pull request by tarka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +37,9 @@ It builds the frontend and puts it in the watch mode with hot reload.
 In other words, if you have the whole plugin and an instance already working, 
 this will enable you to make quick changes with instant preview.
 
-Go to localhost:3333 to run the frontend without Jira
+Go to `http://localhost:3333` to run the frontend without Jira
 
-If you want to test in-app, run atlas-run --product \[jira|refapp|confluence\] while the dev server is running
+If you want to test in-app, run `atlas-run --product [jira|refapp|confluence]` while the dev server is running
```
Comment (theghostwhoforks) [inline README.md]: or `mvn amps:run -Dproduct=jira|refapp|confluence`

File: README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Build the plugin via `mvn package` then install the plugin into your Jira instan
 ### Dependencies
 
 * [Maven](https://maven.apache.org/) (3.6.1 or higher)
+* [Atlassian SDK])(https://developer.atlassian.com/server/framework/atlassian-sdk/)
```
Comment (theghostwhoforks) [inline README.md]: We can call this out as an optional dependency, corrrect?

Comment (tarka) [reply README.md]: Possibly? How likely is it that  someone will do development without the SDK?

Comment (theghostwhoforks) [reply README.md]: I was running this for the longest time without the SDK - using just mvn :)

**Comment by bpar476** [comment]
Looks good! I would just add one note that if you want to use the Atlassian plugin SDK you will need to set your atlas maven version to 3.6.1 or higher:
`export ATLAS_MVN=$(which mvn)`

File: README.md (hunk 1 above)
Comment (nanux) [reply README.md]: I would actually recommend avoiding `atlas-run`, it brings more problems with `mvn` version than necessary.

**Comment by bpar476** [comment]
Should we just drop the `atlas-run` documentation and merge this PR? @tarka

**Comment by tarka** [comment]
> Should we just drop the `atlas-run` documentation and merge this PR? @tarka

It's already been dropped, I'll merge.

**Pull request merged**